ote/test/clj/ote/tasks/pre_notices_test.clj

```diff
@@ -35,6 +35,19 @@
 (defn- email-content []
   (get-in @outbox [0 :body 0 :content]))
 
+(defn- clean-up-db [db]
```

Comment:
Shouldn't the package id be `def`'d in the test, and the test should pass the `def` as an argument to all the db CRUD functions?